src/include/sof/lib/dai.h

```diff
@@ -417,6 +420,18 @@ static inline int dai_get_fifo(struct dai *dai, int direction,
 	return ret;
 }
 
+/**
+ * \brief Get Digital Audio dma srcid
+ */
+static inline int dai_get_srcid(struct dai *dai, int direction,
+			       int stream_id)
+{
+	int ret = dai->drv->ops.get_srcid(dai, direction, stream_id);
+
+
+	return ret;
+}
+
 /**
  * \brief Digital Audio interface Probe
```

Comment:
This patch should be much earlier in the series so that we avoid breaking the bisection. E.g after applying each patch the code should still compile.